src/Components/Publishing/Sections/Sections.tsx

```diff
@@ -109,54 +107,48 @@ export class Sections extends Component<Props, State> {
       ReactDOM.render(<DisplayPanel />, displayMountPoint)
     } else {
       console.error(
-        '(reaction/Sections.tsx) Error mounting Display: DOM node ',
-        'not found', displayMountPoint
+        "(reaction/Sections.tsx) Error mounting Display: DOM node ",
+        "not found",
+        displayMountPoint
       )
     }
   }
 
   getContentStartIndex = () => {
-    const {
-      article: {
-        layout,
-        sections
-      }
-    } = this.props
+    const { article: { layout, sections } } = this.props
 
-    if (layout === 'feature') {
-      const firstText = sections.findIndex(
-        (section) => {
-          return section.type === 'text'
-        }
-      )
+    if (layout === "feature") {
+      const firstText = sections.findIndex(section => {
+        return section.type === "text"
+      })
       return firstText
     }
   }
 
   getSection(section, index) {
+    const { article } = this.props
+
     const sections = {
       image_collection: (
         <ImageCollection
           sectionLayout={section.layout}
+          articleLayout={article.layout}
```

Comment:
This is actually all Prettier changes except this line -- we're now passing down the `articleLayout` again for the other Article types that use this more generic `Sections` component.